lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -737,11 +737,7 @@ enum class StructExtractDifferentiationStrategy {
   // that is zero except along the direction of the corresponding field.
   //
   // Fields correspond by matching name.
-  Fieldwise,
-
-  // Differentiate the `struct_extract` by looking up the corresponding getter
-  // and using its VJP.
-  Getter
+  Fieldwise
```

Comment:
This whole enum can be nuked now.

Reply:
What about when a struct extract instruction is not active?
```swift
auto &strategies = context.getStructExtractDifferentiationStrategies();
// Special handling logic only applies when the `struct_extract` is active.
// If not, just do standard cloning.
if (!activityInfo.isActive(sei, getIndices())) {
  LLVM_DEBUG(getADDebugStream() << "Not active:\n" << *sei << '\n');
  strategies.insert(
      {sei, StructExtractDifferentiationStrategy::Inactive});
  SILClonerWithScopes::visitStructExtractInst(sei);
  return;
}
```

Reply:
Anything directly involving strategies can be nuked I believe.